print.js

```diff
@@ -0,0 +1,40 @@
+var CarLot = (function(cars) {
+    var currentCar;
+
+    //  events if/else statements. 
+    carContainer.addEventListener("click", selectCar);
+
+    function selectCar(e) {
+        var cars = document.getElementsByClassName("container");
+        console.log(container)
+        for (var j = 0; j < cars.length; j++) {
+            cars[j];
+        }
+
+        if (e.target.classList.contains("container")) {
+            e.target.style.border = "solid black 5px";
```

Comment:
Avoid inline styling like this. Try putting these in a CSS class and adding the class to the target instead.